src/pyshark/tshark/output_parser/tshark_ek.py

```diff
@@ -28,8 +28,16 @@ def _extract_packet_from_data(self, data, got_first_packet=True):
 
         return data[start_index:linesep_location], data[linesep_location + 1:]
 
-def packet_from_ek_packet(json_pkt):
-    pkt_dict = orjson.loads(json_pkt)
+def packet_from_ek_packet(json_pkt, allow_invalid_characters=True):
```

Comment:
Right now it essentially _will_ always allow invalid chars.
I thought it might be good to give the option to be more strict if needed.
However, I can remove the flag.

Reply:
i'd remove it, as this is a fork anyway.
if you think it is always preferred, lets just make it default in the code.